src/Examples/UnitModels/Reactors/stoichiometric_reactor_testing.ipynb

```diff
@@ -0,0 +1,1073 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For this operating cost, what conversion did we achieve of ethylene oxide to ethylene glycol? "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 23,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\n",
+      "====================================================================================\n",
+      "Unit : fs.R101                                                             Time: 0.0\n",
+      "------------------------------------------------------------------------------------\n",
+      "    Unit Performance\n",
+      "\n",
+      "    Variables: \n",
+      "\n",
+      "    Key                  : Value       : Fixed : Bounds\n",
+      "               Heat Duty : -5.6566e+06 : False : (None, None)\n",
+      "    Reaction Extent [R1] :      46.400 : False : (None, None)\n",
+      "\n",
+      "------------------------------------------------------------------------------------\n",
+      "    Stream Table\n",
+      "                                                 Inlet     Outlet  \n",
+      "    Molar Flowrate ('Liq', 'ethylene_oxide')      58.000     11.600\n",
+      "    Molar Flowrate ('Liq', 'water')               239.60     193.20\n",
+      "    Molar Flowrate ('Liq', 'sulfuric_acid')      0.33401    0.33401\n",
+      "    Molar Flowrate ('Liq', 'ethylene_glycol') 2.0000e-05     46.400\n",
+      "    Temperature                                   328.15     328.15\n",
+      "    Pressure                                  1.0000e+05 1.0000e+05\n",
+      "====================================================================================\n",
+      "\n",
+      "Conversion achieved =  80.0 %\n"
+     ]
+    }
+   ],
+   "source": [
+    "m.fs.R101.report()\n",
+    "\n",
+    "print()\n",
+    "print('Conversion achieved = ', value(m.fs.R101.conversion)*100, '%')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 24,
+   "metadata": {
+    "tags": [
+     "testing",
+     "remove_cell"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "assert value(m.fs.R101.conversion) == pytest.approx(0.8000, abs=1e-3)\n",
+    "assert value(m.fs.R101.heat_duty[0])/1E6 == pytest.approx(-5.6566, abs=1e-3)\n",
+    "assert value(m.fs.R101.outlet.temperature[0])/1E2 == pytest.approx(3.2815, abs=1e-3)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Optimizing Ethylene Glycol Production\n",
+    "\n",
+    "Now that the flowsheet has been squared and solved, we can run a small optimization problem to minimize our production costs. Suppose we require at least 200 million pounds/year of ethylene glycol produced and 90% conversion of ethylene oxide, allowing for variable reactor volume (considering operating/non-capital costs only) and reactor temperature (heater outlet)."
```

Comment:
The mention of reactor volume here seems out of place, as a stoichiometric reactor by its nature does not depend on volume. I am guessing this is left over from the previous examples.